Sources/API/Debug.swift

```diff
@@ -8,6 +8,12 @@ public class Debug {
     private init() {}
 }
 
+private func withPrint(f: () -> String) -> String {
```

Comment:
I think the folloing is natural
`print(with f: ()-> Stirng)`

Reply:
Thanks for proposal.

But this function name a `withPrint` is intended, add side effects `print()` to another function (`f`) that return `String`. Which **not** means `print()` with another function. In other words, it means "call function and then return `String` **with `print()`**".

Also this function is designed to DSL like because it designed to naturally in call trailing closure style. If adopt `print(with f: () -> String)` like following code that not naturally when reading code.

```swift
print {
    Pretty(formatter: SimpleFormatter()).string(target, debug: false)
}

// or

print(with: {
    Pretty(formatter: SimpleFormatter()).string(target, debug: false)
})
```

Therefore, I think `withPrint()` is naturally than `print(with:)`.

Reply:
I make sense👍